app/src/main/res/layout/activity_contract_detail.xml

```diff
@@ -8,18 +8,15 @@
     android:orientation="vertical"
     tools:context=".feature.dashboard.ui.contractdetail.ContractDetailActivity">
 
-    <androidx.appcompat.widget.Toolbar
-        android:id="@+id/toolbar"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        app:navigationIcon="@drawable/ic_close"
-        app:title="@string/CONTRACT_DETAIL_MAIN_TITLE" />
-
     <androidx.core.widget.NestedScrollView
         android:id="@+id/scrollView"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:layout_marginTop="?actionBarSize">
+        android:layout_marginTop="?actionBarSize"
```

Comment:
If we're using a `ConstraintLayout` as parent, we do not need to use margins to offset the scrollview from the toolbar, we can use constraints instead